spec/requests/api/visitor_can_see_details_of_an_article_spec.rb

```diff
@@ -2,10 +2,11 @@
 
 RSpec.describe Api::ArticlesController, type: :request do
   describe 'GET /article/1 successfully' do
-    let!(:article) { create(:article, title: 'Article Title', lead: 'At some point there will be something the read in the lead', content: 'Article content will go here for the user to read.') }
-    before do
-      get "/api/articles/#{article.id}"
-    end
+    let!(:article) { create(:article, :with_image, title: 'Article Title', lead: 'At some point there will be something the read in the lead', content: 'Article content will go here for the user to read.', premium: true) }
+
+     before do  
+       get "/api/articles/#{Article.last.id}"
```

Comment:
does this need to be capitalized (Article)?

Reply:
It doesn't work with a lower case a, I think it is related to what is calling on in the controller but not 100% sure. Will need to rewatch the demo video from Oliver to confidently answer why it is capitalised.